Classes/ObjectExplorers/Sections/Shortcuts/FLEXShortcutsFactory+Defaults.m

```diff
@@ -25,7 +25,7 @@ + (void)load { FLEX_EXIT_IF_NO_CTORS()
     FLEXRuntimeUtilityTryAddObjectProperty(
         2, sharedApplication, UIApplication.flex_metaclass, UIApplication, PropertyKey(ReadOnly)
     );
-    
```

Comment:
Well, my editor auto removes all these spaces, feel free to revert these changes if you prefer to do so.

Reply:
Yeah, I'll do this in the morning 😅

For what it's worth, if you use Git Tower, it's really easy to just commit the functional changes and then discard all the leftover whitespace changes! I haven't used other apps so idk how well they handle line-by-line changes, and I know it's a pain if you're just using the command line